Makefile

```diff
@@ -16,8 +16,15 @@ CONTROL_KEYSPACE ?= OTTER
 REPLICATION_FACTOR ?= 3
 CLOUDCAFE ?= $(shell which cafe-runner)
 
+mkfile_dir := $(shell dirname "$(MAKEFILE_LIST)")
+
 .PHONY: targets env-precheck
 
+hooks:
+	cp ${mkfile_dir}/scripts/config_check.py ${mkfile_dir}/.git/hooks
+	echo "#/bin/bash" > ${mkfile_dir}/.git/hooks/pre-commit
+	echo "python .git/hooks/config_check.py" >> ${mkfile_dir}/.git/hooks/pre-commit
```

Comment:
I might just replace this with a `ln -s` and put a `#!/usr/bin/env python` shebang and executable flag on config_check.py itself; fewer files to keep updated.